app/src/main/java/com/duckduckgo/app/global/store/AndroidUserBrowserProperties.kt

```diff
@@ -82,6 +82,6 @@ class AndroidUserBrowserProperties(
     }
 
     override suspend fun networkProtectionOnboarded(): Boolean {
-        return networkProtectionState.isOnboarded()
+        return networkProtectionState.isOnboarded() || networkProtectionState.wasEnabledDuringBeta()
```

Comment:
@aitorvs this is related to the problem I mentioned over MM re: isOnboarded being always false once beta ends